operator/test/test.sh

```diff
@@ -97,56 +97,52 @@ wait_for_pipeline() {
 }
 
 check_pod_security() {
-  sccs="$(kubectl get pod -o name -n "$1" | xargs -l -r kubectl get -o jsonpath='{.metadata.annotations.openshift\.io/scc}' -n "$1")"
-  if [[ "$sccs" =~ "restricted-v2" ]]; then
-     prune="$(echo "$sccs" | sed 's/restricted-v2//g')"
-     # if anything besides restricted-v2 is in there, we want to investigate
-     if [ -z "$prune" ]; then
-       echo "   - OK pod security for $1"
-     else
-       echo "Failed, scc's are "
-       echo "$sccs"
-       echo "[ERROR] Unexpected $1 pod security context constraints" >&2
-       exit 1
-     fi
-  else
-     # if none of the pods are restricted-v2, we want to investigate
-     echo "Failed, scc's are "
-     echo "$sccs"
-     echo "[ERROR] Unexpected $1 pod security context constraints" >&2
-     exit 1
+  mapfile -t pods < <(kubectl get pod -o name -n "$1")
+  if [[ "${#pods[@]}" -gt 0 ]]; then
+    for pod in "${pods[@]}"; do
+      if [[ "$pod" == "pod/storage-pool-0-0" ]] || [[ "$pod" == "pod/postgres-postgresql-0" ]]; then
+        echo "   - OK skipping $pod as it is exempt"
+        continue
+      fi
+      scc="$(kubectl get "$pod" -o jsonpath='{.metadata.annotations.openshift\.io/scc}' -n "$1"  --ignore-not-found)"
+      if [ -z "$scc" ]; then
+        echo "   - OK pod security check for $pod skipped (most like the pod terminated quickly)"
```

Comment:
`    - Skip: $pod could not be inspected (most likely because it was terminated quickly)`